java/CI/generate-sdk.sh

```diff
@@ -4,11 +4,34 @@ set -e
 BASE_PATH=/java # Base location of Java generated classes
 PACKAGE_PATH=src/main/java
 
+BASE_PATH_v1 = /java/v1
+BASE_PATH_v2 = /java/v2
+BASE_PATH_v3 = /java/v3
+
 rm -f $BASE_PATH/$PACKAGE_PATH/**/*.java
```

Comment:
You can probably do `rm -f $BASE_PATH/**/*.java` to get cover all schema packages. Or get rid of this one in favor of line s13-15.